.github/workflows/test.yaml

```diff
@@ -0,0 +1,22 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v1
```

Comment:
Any reason we're using v1 over the latest v2?

Related, is it worth [enabling caching for dependencies](https://github.com/actions/setup-node#caching-packages-dependencies)?

Reply:
Good catch!